src/Microsoft.Health.Fhir.CosmosDb.UnitTests/Features/Operations/Export/ExportOrchestratorJobTests.cs

```diff
@@ -0,0 +1,268 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export.Models;
+using Microsoft.Health.Fhir.Core.Features.Search;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.CosmosDb.Features.Operations.Export;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.JobManagement;
+using Microsoft.Health.Test.Utilities;
+using Newtonsoft.Json;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.CosmosDb.UnitTests.Features.Operations.Export
+{
+    [Trait(Traits.OwningTeam, OwningTeam.Fhir)]
+    [Trait(Traits.Category, Categories.Export)]
+    public class ExportOrchestratorJobTests
```

Comment:
This should be CosmosExportOrchestratorJobTests

Reply:
Will change SQL as well - initially was trying to mirror SQL tests.